src/js/components/ConfigurationView.js

```diff
@@ -0,0 +1,242 @@
+import mixin from 'reactjs-mixin';
+import React from 'react';
+import {StoreMixin} from 'mesosphere-shared-reactjs';
+
+import DCOSStore from '../stores/DCOSStore'
+import DescriptionList from './DescriptionList';
+import Service from '../structs/Service';
+import StringUtil from '../utils/StringUtil';
+
+const sectionClassName = 'container-fluid container-pod container-pod-super-short flush flush-bottom';
+
+function fetchVersion(service, versionID) {
+  if (service.getVersions().get(versionID) == null) {
+    DCOSStore.fetchServiceVersion(service.getId(), versionID);
+  }
+}
+
+class ConfigurationView extends mixin(StoreMixin) {
+  constructor() {
+    super(...arguments);
```

Comment:
The `...arguments` kinda makes sure, that this component is future proof and we're not missing something if we update one of the parent classes. 
I don't think that we have a rule for these yet, thus I suggest to discuss this in our next sync meeting. 

/cc @rcorral /link https://github.com/dcos/dcos-ui/pull/207#discussion_r63930608